src/moduler/aktivitet/visning/avtalt-container/SattTilAvtaltInfotekst.tsx

```diff
@@ -7,10 +7,11 @@ import { useKanSendeVarsel } from './avtaltHooks';
 const getText = (
     kanSendeVarsel: boolean,
     avtaltMedNavMindreEnnSyvDager: boolean,
+    manglerTilDato: boolean,
```

Comment:
Foretrekker egentlig å gjøre boolske verdier positive per default, altså at man snur denne til "harTilDato" fordi koden ofte da blir mer lesbar, spesielt hvis man nå, eller senere får behov for å sjekke at noe `!manglerTilDato`. Da må jeg tenke meg om en ekstra gang når man sitter med dobbel negasjon